src/domain/fx/cyril.js

```diff
@@ -82,20 +83,23 @@ const getParticipantAndCurrencyForFxTransferMessage = async (payload) => {
   // Does this determining transfer ID appear on the transfer list?
   const transferRecord = await TransferModel.getById(payload.determiningTransferId)
   const determiningTransferExistsInTransferList = (transferRecord !== null)
-
   let participantName, currencyId, amount
 
   if (determiningTransferExistsInTransferList) {
     // If there's a currency conversion which is not the first message, then it must be issued by the creditor party
     // Liquidity check and reserve funds against FXP in FX target currency
-    participantName = payload.counterPartyFsp
-    currencyId = payload.targetAmount.currency
-    amount = payload.targetAmount.amount
-    await watchList.addToWatchList({
-      commitRequestId: payload.commitRequestId,
-      determiningTransferId: payload.determiningTransferId,
-      fxTransferTypeId: Enum.Fx.FxTransferType.PAYEE_CONVERSION
-    })
+    throw ErrorHandler.Factory.createFSPIOPError(
+      ErrorHandler.Enums.FSPIOPErrorCodes.NOT_IMPLEMENTED,
+      'Payee FX conversion not implemented'
+    )
+    // participantName = payload.counterPartyFsp
+    // currencyId = payload.targetAmount.currency
+    // amount = payload.targetAmount.amount
+    // await watchList.addToWatchList({
+    //   commitRequestId: payload.commitRequestId,
+    //   determiningTransferId: payload.determiningTransferId,
+    //   fxTransferTypeId: Enum.Fx.FxTransferType.PAYEE_CONVERSION
+    // })
```

Comment:
I think payee side currency conversion is already implemented by seeing this code. I don't think we need to disable that.

Reply:
But then we would need to give it the whole suite of testing, no? I thought testing payee conversion was out of scope? @vijayg10